actiontext/lib/templates/installer.rb

```diff
@@ -29,4 +29,14 @@
       append_to_file APPLICATION_PACK_PATH, "\n#{line}"
     end
   end
+else
+  warn <<~WARNING
+    WARNING: In order to use Action Text, application should either include webpacker gem(https://github.com/rails/webpacker)
```

Comment:
Is following a better warning message?

```
WARNING: Action Text requires `trix` and `@rails/actiontext` JavaScript dependencies to be added to the bundles. Your application
is missing the requirement.

To fix this, you can either add webpacker gem(https://github.com/rails/webpacker) and follow the installation guide to add webpack and
re-run the task `bin/rails action_text:install` to add JavaScript dependencies.

OR add following lines to the JavaScript bundles directly.

require("trix")
require("@rails/actiontext")

```

Reply:
I prefer the text in your comment 👍

Reply:
I still think the message could use a little tuning.

- "is missing the requirement" sounds lightly anxiety inducing, users haven't done anything wrong so let's instead state what Action Text can't do automatically.
- Let's move the manual instructions up such that people don't get lost in any Webpacker setup if they just want to start using Action Text.
- The Webpacker instructions seem more detailed than is needed in this warning.

Here's how I think that could sound:

```
WARNING: Action Text can't locate your JavaScript bundle to add its package dependencies.

Add these lines to any bundles:

require("trix")
require("@rails/actiontext")

Alternatively install and setup the webpacker gem then rerun `bin/rails action_text:install`
to have these dependencies added automatically.
```